api/validators/application.py

```diff
@@ -14,7 +14,7 @@
         "name": {
             "type": "string",
         },
-        "avatar": {
+        "profile-pic": {
```

Comment:
For JSON related stuff, using camel case is the standard! So instead of profile-pic, you would use profilePic - please make the changes for the following!